src/main/java/uk/gov/crowncommercial/dts/scale/cat/controller/EventsController.java

```diff
@@ -171,4 +171,20 @@ public StringValueResponse publishEvent(@PathVariable("procID") final Integer pr
     return new StringValueResponse("OK");
   }
 
+  @GetMapping("/{eventID}/messages")
+  public Collection<MessageSummary> getMessages(
+          @PathVariable("procID") final Integer procId,
+          @PathVariable("eventID") final String eventId,
+          @RequestParam(name = "message-direction",required = false) String messageDirection,
+          @RequestParam(name = "message-read",required = false) String messageRead,
+          @RequestParam(name = "sort") String sort,
```

Comment:
These can be deserialized directly to the enum types I believe (as the generated types have `@JsonCreator` on their fromValue method - e.g. `MessageSort` - so should just work

Reply:
done